### DISCUSSION TITLE
[xstate/inspect] official chrome extension which inspect can connect to as an option

### DISCUSSION THREAD

**Discussion by ilovett** [statelyai/xstate#1459] [General]
Add an option to serve the inspector from within a simple chrome dev tools extension wrapper instead of iframe or new tab so we can get window resizing on the same window.

```
inspect({ mode: 'extension' })
```

**Comment by davidkpiano** [comment]
Let's post this here: https://github.com/davidkpiano/xstate/discussions/1430
